index.bs

```diff
@@ -3537,19 +3537,17 @@ This attestation statement format is used with FIDO U2F authenticators using the
         key over the P-256 curve, terminate this algorithm and return an appropriate error. 
     1. Extract the claimed |rpIdHash| from |authenticatorData|, and the claimed |credentialId| and |credentialPublicKey| from
         |authenticatorData|.<code>[=attestedCredentialData=]</code>.
-    1. Convert the COSE_KEY formatted |credentialPublicKey| (see Section 7 of [[!RFC8152]]) to CTAP1/U2F public Key
-        format [[!FIDO-CTAP]]. 
-        - Let |publicKeyU2F| represent the result of the 
-            conversion operation and set its first byte to 0x04.
-            Note: This signifies uncompressed ECC key format.
-        - Extract the value corresponding to the "-2" key 
-            (representing x coordinate) from |credentialPublicKey|, 
-            confirm its size to be of 32 bytes and concatenate it with |publicKeyU2F|.
+    1. Convert the COSE_KEY formatted |credentialPublicKey| (see Section 7 of [[!RFC8152]]) to CTAP1/U2F user public key
+        format (see Section 4.3 of [[!FIDO-U2F-Message-Formats]]).
```

Comment:
We can link directly to both Section 7 of [[!RFC8152]] and Section 4.3 of [[!FIDO-U2F-Message-Formats]], tho see issue #872 (i would make entries in `<pre class="anchors">` to do so).

Reply:
Ok. I made an attempt at this in PR #883, does that look okay?

Reply:
@emlun maybe "1. Convert the COSE_KEY formatted |credentialPublicKey| (see Section 7 of [[!RFC8152]]) to Raw ANSI X9.62 public key format(see ALG_KEY_ECC_X962_RAW  in Section 3.6.2 of [[!FIDO-Registry]]).

Reply:
@emlun 
> k. I made an attempt at this in PR #883, does that look okay?
yes, see https://github.com/w3c/webauthn/pull/883#pullrequestreview-116424391

Reply:
@herrjemand's suggestion is also fine by me and we can use the same technique as in #883 to link directly to Section 3.6.2 of [[!FIDO-Registry]]).  either works for me.

Reply:
`[[!FIDO-Registry]]` doesn't seem to be defined in the spec yet - what does it refer to?

Reply:
@emlun To this https://drafts.fidoalliance.org/fido-2/latest/fido-registry-v2.0-wd-20180426.html *)